src/entities/studentsData/studentsData.entity.ts

```diff
@@ -1,10 +1,14 @@
-import { Entity, Column, BaseEntity, PrimaryColumn } from 'typeorm';
+import { Entity, Column, BaseEntity, PrimaryColumn, OneToOne, JoinColumn } from 'typeorm';
 import { ContractType, StudentStatus, StudentsDataInterface, choiceYesNO, expectedTypeWorkEntity } from '../types/studentsData';
+import { User } from '../User/User.entity';
 
 @Entity('students_data')
 export class StudentsData extends BaseEntity implements StudentsDataInterface {
   @PrimaryColumn()
   id: string;
+  @OneToOne(() => User, { 'cascade': true })
```

Comment:
the same here, enter + primary:true

Reply:
Robiłem testy i przy tym zapisie jest ok.